docs/docs/cmd/spo/field/field-list.md

```diff
@@ -0,0 +1,42 @@
+# spo field list
+
+Retrieves list of columns for specified list or site
+
+## Usage
+
+```sh
+m365 spo field list [options]
+```
+
+## Options
+
+`-u, --webUrl <webUrl>`
+: Absolute URL of the site where the fields is located
+
+`-t, --listTitle [listTitle]`
+: Title of the list where the fields are located. Specify only one of listTitle, listId or listUrl
+
+`-i --listId [listId]`
+: ID of the list where the fields are located. Specify only one of listTitle, listId or listUrl
+
+--8<-- "docs/cmd/_global.md"
+
+## Examples
+
+Retrieves list columns for _https://contoso.sharepoint.com/sites/contoso-sales_.
```

Comment:
Wouldn't this command be retrieving site columns as we only specify the site URL without any list?

Reply:
yes it will. Thanks for noticing this one 👍